en-US/snippets/2018/Firefox100-part1.lang

```diff
@@ -0,0 +1,295 @@
+
+
+## MAX_LENGTH: 180
+# Limit character length to 180, preferably under 160. Please rephrase as needed to fit.
+# Campaign: Add-ons
+# Link: https://addons.mozilla.org/firefox/
+# Week 1
+;Personalize Firefox with Add-ons for privacy, social, gaming, and a whole lot more. <a>Make Firefox yours</a>.
+Personalize Firefox with Add-ons for privacy, social, gaming, and a whole lot more. <a>Make Firefox yours</a>.
+
+
+## MAX_LENGTH: 180
+# Limit character length to 180, preferably under 160. Please rephrase as needed to fit.
+# Campaign: Delight
+# Week 1
+;Fun fact: Firefox users are the smartest, funniest, best-looking people on the Web. [Citation needed]
+Fun fact: Firefox users are the smartest, funniest, best-looking people on the Web. [Citation needed]
+
+
+## MAX_LENGTH: 180
+# Limit character length to 180, preferably under 160. Please rephrase as needed to fit.
+# Campaign: Internet Health
+# Link: https://www.mozilla.org/internet-health/
+# This message targets new users. "Webby journeys" mean all the browsing on the internet.
+# Week 1
+;Hey, nice work choosing Firefox for your webby journeys. Not all heroes wear capes and by browsing with us, <a>you're helping build a better internet</a>.
+Hey, nice work choosing Firefox for your webby journeys. Not all heroes wear capes and by browsing with us, <a>you're helping build a better internet</a>.
+
+
+## MAX_LENGTH: 180
+# Limit character length to 180, preferably under 160. Please rephrase as needed to fit.
+# Campaign: SUMO
+# Link: https://support.mozilla.org/kb/get-started-firefox-overview-main-features
+# Week 1
+;Whether you’re new to Firefox or just need a refresher, here’s a quick guide to all the <a>awesomeness Firefox comes with right out of the box</a>.
+Whether you’re new to Firefox or just need a refresher, here’s a quick guide to all the <a>awesomeness Firefox comes with right out of the box</a>.
+
+
+## MAX_LENGTH: 180
+# Limit character length to 180, preferably under 160. Please rephrase as needed to fit.
+# Campaign: Firefox Mobile
+# Week 2
+;Browse without boundaries. Send Firefox to your mobile device for better browsing on the go.
+Browse without boundaries. Send Firefox to your mobile device for better browsing on the go.
+
+
+## MAX_LENGTH: 180
+# Limit character length to 180, preferably under 160. Please rephrase as needed to fit.
+# Campaign: Firefox Mobile
+# Week 2
+;Browse unbound. Send Firefox to your phone and take a powerful independent browser with you.
+Browse unbound. Send Firefox to your phone and take a powerful independent browser with you.
+
+
+## MAX_LENGTH: 180
+# Limit character length to 180, preferably under 160. Please rephrase as needed to fit.
+# Campaign: Firefox Mobile
+# Week 2
+;Full-featured. Customizable. Lightning fast. Browse without compromise with Firefox Mobile.
+Full-featured. Customizable. Lightning fast. Browse without compromise with Firefox Mobile.
```

Comment:
Isn't this missing a piece?

Reply:
The statement is copied from here: https://www.mozilla.org/en-US/firefox/mobile/

Reply:
So weird, when looking at the spreadsheet I saw another sentence after this in that snippet.